src/esssans/isis/__init__.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Should the `sans2d` submodule be part of `isis`?

Reply:
Probably, but I want to keep any unrelated changes out of this PR. We can perform such restructuring and rollout of changes in a follow-up.